socorro/processor/signature_utilities.py

```diff
@@ -610,7 +610,10 @@ def version(self):
         return '1.0'
 
     def _predicate(self, raw_crash, raw_dumps, processed_crash, proc_meta):
-        return processed_crash['signature'].startswith('EMPTY')
+        return bool(
+            processed_crash['signature'].startswith('EMPTY') and
+            processed_crash.get('mdsw_status_string')
+        )
```

Comment:
This adjusts the predicate so it only returns true iff the signature starts with `EMPTY` and there's an `mdsw_status_string` value. The figuring is that if we don't have a value, then there's nothing to concatenate in the action anyhow.